daemon/ct.go

```diff
@@ -65,21 +65,23 @@ func (d *Daemon) EnableConntrackGC() {
 		for {
 			sleepTime := time.Duration(GcInterval) * time.Second
 
-			d.endpointsMU.Lock()
+			d.endpointsMU.RLock()
 
 			for k := range d.endpoints {
 				e := d.endpoints[k]
+				e.Mutex.RLock()
 				if e.Consumable == nil {
+					e.Mutex.RUnlock()
 					continue
 				}
-
+				e.Mutex.RUnlock()
```

Comment:
I add this:
+                               // We can unlock the endpoint mutex sense
+                               // in runGC it will be locked as needed.
Sounds sufficient?